src/hooks/use-web3-call-state.ts

```diff
@@ -0,0 +1,127 @@
+import { useState } from "react";
+import { Web3Authentication, web3Prove } from "../components/web3/api";
+import { POAP, NFTcollection } from "../components/web3/core";
+import { generateRoomName } from "../lib";
```

Comment:
Is this function still available? I'm not seeing it in `src/lib.ts` which is where this appears to be pointing to at the moment.

Reply:
It is, here's the relevant lines: https://github.com/brave/brave-talk/blob/web3-on-reactify/src/lib.ts#L26-L34

Reply:
Ahh thanks, I wasn't reading the source tree properly.